```
2022-08-24T11:05:25.3485601Z C:\Users\VssAdministrator\.nuget\packages\microsoft.typescript.msbuild\4.7.4\tools\tsc\lib.dom.d.ts(5297,14): error TS2687: Build:All declarations of 'kind' must have identical modifiers. [D:\a\1\s\src\Web\Avalonia.Web.Blazor\Avalonia.Web.Blazor.csproj]
2022-08-24T11:05:25.3490253Z D:\a\1\s\src\Web\Avalonia.Web.Blazor\Interop\Typescript\StorageProvider.ts(11,9): error TS2687: Build:All declarations of 'kind' must have identical modifiers. [D:\a\1\s\src\Web\Avalonia.Web.Blazor\Avalonia.Web.Blazor.csproj]
2022-08-24T11:05:25.3492711Z D:\a\1\s\src\Web\Avalonia.Web.Blazor\Interop\Typescript\StorageProvider.ts(11,9): error TS2717: Build:Subsequent property declarations must have the same type.  Property 'kind' must be of type '"file"', but here has type '"file" | "directory"'. [D:\a\1\s\src\Web\Avalonia.Web.Blazor\Avalonia.Web.Blazor.csproj]
2022-08-24T11:05:25.3495034Z D:\a\1\s\src\Web\Avalonia.Web.Blazor\Interop\Typescript\StorageProvider.ts(172,13): error TS2367: Build:This condition will always return 'true' since the types '"file"' and '"directory"' have no overlap. 
```

--- a/src/Web/Avalonia.Web.Blazor/Interop/Typescript/StorageProvider.ts
+++ b/src/Web/Avalonia.Web.Blazor/Interop/Typescript/StorageProvider.ts
@@ -8,7 +8,6 @@ declare global {
     type PermissionsMode = "read" | "readwrite";
     interface FileSystemFileHandle {
         name: string,
-        kind: "file" | "directory",
         getFile(): Promise<File>;
         createWritable(options?: { keepExistingData?: boolean }): Promise<FileSystemWritableFileStream>;
 
@@ -169,10 +168,6 @@ class StorageItem {
     }
 
     public async getItems(): Promise<StorageItems> {
-        if (this.handle.kind !== "directory"){
-            return new StorageItems([]);
-        }
-        
         const items: StorageItem[] = [];
         for await (const [key, value] of this.handle.entries()) {
             items.push(new StorageItem(value));

